### ISSUE TITLE
Connection signage causes build error

### ISSUE THREAD

**Issue by markd0811** [BEEmod/BEE2-items#3077]
Item: Connection Signage
Item thumbnail: 
![image](https://user-images.githubusercontent.com/28679079/61987374-eca4f900-afca-11e9-80cb-d9e1a19c40f3.png)

Having this item in a map causes portal 2 to abort building geometry and show a build error message.
Tested with "Build and Play Puzzle" button only, not "Publish"
Tested with both "Fast" and "Full" lighting in beemod compile settings
Tested with both "Clean" and "Overgrown" themes.
Tested with item values set to "--," "03," "04."
Problem in release 35.1, but not in 34 or below.

As soon as every instance of this item is deleted from the editor, building works again.

Thank you for all your work on this project!

**Comment by JavierNovellon** [comment]
This also happens to me. I can't remember if it's ALL of the styles (I'll check) but it most recently happened with "Behind The Scenes". Exactly as you're describing it.
Either hitting F9 or the button on the toolbar, Portal 2 gives an error message.

PS - The publishing process does indeed have the same result, since it has to rebuild the chamber as normal before uploading the file to the Steam Workshop.

**Comment by Konclan** [comment]
Still a problem

**Comment by Luke18033** [comment]
Seems to be a missing instance:
```
[I] <valve>.vbsp(): Could not open instance file instances/BEE2/logic/BEE/antline_signage.vmf
```

**Comment by Determinated888** [comment]
Still an issue as of 4.36.1

**Comment by executionByFork** [comment]
This is still an issue and has not been fixed as of 4.36.1 using 4.36.0 packages. No matter the options used on the connection signage, it errors out during build. I've tried with all timer values and checking/unchecking reverse and enabled booleans.

It looks like the commit [ec8ab72](https://github.com/BEEmod/BEE2.4/commit/ec8ab725edd2a7bcf44f494dd33332e2f36a1a79) fixes this... Is there a schedule for the next release to be pushed out with this and other fixes?
